terraform/deployments/103495720024_exec_roles/variables.tf

```diff
@@ -14,4 +14,9 @@ variable "splunk_forwarder_name" {
   description = "Name used to derive a role ARN and path prefix for S3."
   type        = string
   default     = "co-cdio-technology-official-it-platform-wec"
-}
\ No newline at end of file
+}
+
+variable "ssh_key_name" {
+  description = "A name for AWS Keypair to use to auth to helk. Can be anything you specify."
```

Comment:
I find this description a little vague, but I assume this will be clarified when we write a readme